server/request_handler.go

```diff
@@ -59,6 +60,14 @@ func (r *RpcRequestHandler) process() {
 		return
 	}
 
+	var isFast bool
```

Comment:
Nit: I'm guessing this can be declared inside if block

Reply:
Done.

Reply:
it's not needed at all

Reply:
updated